README.md

```diff
@@ -93,6 +93,7 @@ handle it from there. :smile:
   * [Shunting yard](cpp/include/algorithm/string/shunting_yard.hpp) :white_check_mark:
   * Permutation
     * [Heap's Algorithm](cpp/include/algorithm/string/heaps_algorithm.hpp) :white_check_mark:
+  * [Polynomial Rolling Hash Function](cpp/include/algorithm/string/string_hash.hpp) :white_check_mark:  
```

Comment:
My instinct is to change this to "String hashing" as I don't think this level of specificity (nor the implementation details) are necessary here. Thoughts?

Reply:
I agree with this statement, the specific implementation is not important as a hashing algorithm just returns some form of a hash so I will change the name to String Hashing.